src/App.tsx

```diff
@@ -34,62 +35,137 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort((a, b) => a.length - b.length);
+      break;
+
+    default:
```

Comment:
return just visible goods in default

Reply:
https://github.com/mate-academy/react_list-of-goods/pull/1637/files#r1167989440
not fixed